modules/programs/nushell.nix

```diff
@@ -121,6 +121,15 @@ in {
         this option) to command strings or directly to build outputs.
       '';
     };
+
+    environmentVariables = mkOption {
+      type = types.attrsOf types.str;
+      default = { };
+      example = { FOO = "BAR"; };
```

Comment:
Wouldn't `BAR` need to be quoted, if this is interpreted by the shell?

I think it would make more sense to escape the values here so that strings are treated literally, and you can still use `extraEnv` for variables that require arbitrary code.

Reply:
I personally like that both are possible here.
You can use it for arbitrary code if you want to or use it with just bare strings if you prefer.
Do you feel strongly about this ?
For me extraEnv would be for stuff that require intermediary variables or more complicated code. One liners would still fit in the environmentVariables category.

Reply:
> Do you feel strongly about this ?

Not at all. I don't even use nushell.